src/components/Home.jsx

```diff
@@ -1,3 +1,32 @@
+import SingleArticle from "./ArticleCard";
+import * as API from '../utils/api';
+import { useState, useEffect } from 'react';
+
 export default function Home () {
-    
+
+    // States To Track User's Filtering - If any of the sort/order/topic change then re-render the articles displayed
+    const [displayedArticles, setDisplayedArticles] = useState([]); 
+    const [sort, setSort] = useState('created_at');
+    const [order, setOrder] = useState('DESC');
+    const [topic, setTopic] = useState('');
+
+    useEffect(() => {
+        API.getArticles()
+        .then((articlesData) => {
+            console.log(articlesData, '<<Success')
+            setDisplayedArticles(articlesData);
+        })
+    }, [sort, order, topic])
+
+    return(
```

Comment:
this map is working because your default state is an empty array, but we would normally handle this with an isLoading pattern. Also dont forget to handle the errors, what would happen here if your API errored out?